### ISSUE TITLE
Is it confusing to have emails coming from 2 different names?

### ISSUE THREAD

**Issue by Jarrettob** [Attendize/Attendize#239]
Thinking this may be a user issue in the future. Tickets come from Account level and a person's name, then messages are sent from the organizer and another person Wondering if there might be an account level setting to override to the organizer for sending tickets, or just have that be the default.

**Comment by feliphed** [comment]
Hello. I can not install this script. Can you help me?

**Comment by daveearley** [comment]
Yes this should be changed alright, there are a few places where the global 'from' setting is used when it should be the organiser's name.

**Comment by johannac** [comment]
This still needs to be standardized.
